packages/main/src/TableHeaderRow.hbs

```diff
@@ -2,11 +2,11 @@
     <ui5-table-header-cell id="selection-cell"
         aria-selected="{{_isSelected}}"
         aria-label="{{_i18nSelection}}"
+        data-ui5-table-selection-component
     >
     {{#if _isMultiSelect}}
         <ui5-checkbox id="selection-component" tabindex="-1"
             ?checked="{{_isSelected}}"
-            @ui5-change="{{_informSelectionChange}}"

```

Comment:
I am not sure about this change Duc.
To replace this you probably handle this with listening space/enter/click but maybe one day we change from checkbox to other control e.g. switch or maybe the checkbox will fire the change event with another interaction.

If this is not a big change I would rather suggest to keep it there. What do u think?

Reply:
My though process was more or less, that the Selection feature should handle all things related to the selection.
This would also include handling clicks to the check/radioboxes.

Maybe in future it is even possible to define a "selectionCellTemplate" in the TableSelection feature, which could be applied to the rows, without a "direct" dependency to the TableSelection.

But for now, you are right, it's a bit much for the current change, I think. Let's discuss this separately.

Reply:
i am ok with both